Packs/Mimecast/Scripts/MimecastFindEmail/MimecastFindEmail.py

```diff
@@ -0,0 +1,45 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+MAILBOXES_CTXKEY = "Mailboxes"
+
+
+def main() -> None:
+    res = []
+    resp = demisto.executeCommand("mimecast-query", demisto.args())
+    users = set()
```

Comment:
No need to define it here (too early, too much context...)
Move the definition to inside the `if data`